lib/executor/externally_controlled.go

```diff
@@ -341,7 +341,8 @@ type manualVUHandle struct {
 
 func newManualVUHandle(
 	parentCtx context.Context, state *lib.ExecutionState,
-	localActiveVUsCount *int64, initVU lib.InitializedVU, logger *logrus.Entry,
+	localActiveVUsCount *int64, initVU lib.InitializedVU,
+	config *BaseConfig, logger *logrus.Entry,
```

Comment:
nit: maybe have this be a method on *externallyControlledRunState as there are now 5 parameters that are its fields ...

Reply:
Good idea, should be resolved by 438b422e.